CHANGELOG.md

```diff
@@ -1,3 +1,124 @@
+- OpenVPN image to version v2.5.0-r1. [#6697](https://github.com/kubermatic/kubermatic/issues/6697) 
+- machine-controller to v1.27.1 [#6695](https://github.com/kubermatic/kubermatic/issues/6695) 
+- controller-runtime 0.8.1 [#6450](https://github.com/kubermatic/kubermatic/issues/6450) 
+- cert-manager to 1.2.0 [#6739](https://github.com/kubermatic/kubermatic/issues/6739) 
+
+### Dashboard
+
+- Add resource quota settings to the admin panel. [#3019](https://github.com/kubermatic/dashboard/issues/3019) 
+- Add autocompletions for vSphere datastores. [#3020](https://github.com/kubermatic/dashboard/issues/3020) 
+- Add option to disable User SSH Key Agent from the cluster wizard. [#3025](https://github.com/kubermatic/dashboard/issues/3025) 
+- Remove CoreOS [#3027](https://github.com/kubermatic/dashboard/issues/3027) 
+- AWS node sizes in the wizard now provide GPU information. [#3038](https://github.com/kubermatic/dashboard/issues/3038) 
+- Filter external openstack networks during cluster creation [#3053](https://github.com/kubermatic/dashboard/issues/3053) 
+- Add changelog support [#3081](https://github.com/kubermatic/dashboard/issues/3081) 
+- Remove OpenShift support. [#3100](https://github.com/kubermatic/dashboard/issues/3100) 
+- Redesign add/edit member dialog [#3104](https://github.com/kubermatic/dashboard/issues/3104) 
+- Add GPU count display for Alibaba instance types. [#3113](https://github.com/kubermatic/dashboard/issues/3113) 
+- Remove duplicated KubeAdm hints from cluster page. [#3114](https://github.com/kubermatic/dashboard/issues/3114) 
+- Redesign manage SSH keys dialog on cluster details to improve user experience. [#3120](https://github.com/kubermatic/dashboard/issues/3120) 
+- Change VSPhere's diskSizeGB option from optional to required. [#3121](https://github.com/kubermatic/dashboard/issues/3121) 
+- Redesign autocomplete inputs. Right now spinner will be displayed next to the input that loads autocompletions in the background. [#3122](https://github.com/kubermatic/dashboard/issues/3122) 
+- Add info about GPU count for Azure instances. [#3140](https://github.com/kubermatic/dashboard/issues/3140) 
+- Allow custom links to be placed in the help and support panel [#3141](https://github.com/kubermatic/dashboard/issues/3141) 
+- Add support for OPA to UI [#3147](https://github.com/kubermatic/dashboard/issues/3147)
+- Add network to Hetzner [#3158](https://github.com/kubermatic/dashboard/issues/3158)
+- Add `enable` and `enforce` OPA options to Admin Settings [#3206](https://github.com/kubermatic/dashboard/issues/3206)
+
+### Bugfixes
+
+- Fix bug with changing the theme based on the color scheme if enforced_theme was set. [#3163](https://github.com/kubermatic/dashboard/issues/3163)
+
+### Supported Kubernetes Versions
+
+* 1.18.6
+* 1.18.8
+* 1.18.10
+* 1.18.14
+* 1.18.17
+* 1.19.0
+* 1.19.2
+* 1.19.3
+* 1.19.8
+* 1.19.9
+* 1.20.2
+* 1.20.5
+* 1.21.0
```

Comment:
Usually, we've put this information at the beginning of the changelog. Should we move this to the top?

Reply:
sure